tsconfig.base.json

```diff
@@ -10,7 +10,8 @@
       "test_utils/*": [
         "src/test_utils/public/*"
       ],
-      "fixtures/*": ["src/fixtures/*"]
+      "fixtures/*": ["src/fixtures/*"],
+      "@opensearch-project/opensearch": ["node_modules/@opensearch-project/opensearch/api/new"]
```

Comment:
I think this should be fine. If it builds fine. You might not be in a state to `yarn build-platform linux --skip-os-packages --release`, but we should keep this in mine when doing an overall review.